### PR TITLE
events maven upgrade

### PR DESCRIPTION
**Pull request by shreyab30-eGov**

### CODE REVIEW

File: egov-user-event/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this module will be documented in this file.
+
+  ## 1.1.1 - 2020-06-29
```
Comment (tarunlalwani) [inline egov-user-event/CHANGELOG.md:4]: Correct spacing and make it `1.1.0`

File: egov-user-event/src/test/resources/application.properties [hunk 2]
```diff
@@ -5,17 +5,19 @@ spring.datasource.username=postgres
...
 
 #----------------------------- FLYWAY CONFIGURATIONS ------------------------------#
-flyway.url=jdbc:postgresql://localhost:5432/hr_employee_db
-flyway.user=postgres
-flyway.password=postgres
-flyway.table=hr_employee_schema_version
-flyway.baseline-on-migrate=true
-flyway.outOfOrder=true
-flyway.locations=db/migration/main,db/migration/seed
+spring.flyway.url=jdbc:postgresql://localhost:5432/hr_employee_db
```
Comment (tarunlalwani) [inline egov-user-event/src/test/resources/application.properties:8]: This still has hrms stuff?

File: egov-user-event/CHANGELOG.md (hunk 1 above)
Comment (shreyab30-eGov) [reply egov-user-event/CHANGELOG.md]: Updated to CHANGELOG.md

File: egov-user-event/src/test/resources/application.properties (hunk 2 above)
Comment (shreyab30-eGov) [reply egov-user-event/src/test/resources/application.properties:8]: Its updated in this commit  - https://github.com/egovernments/municipal-services/pull/263/commits/c3bcbee99a0cd8a58c5a4d7f21b6b391448bb47f

**Pull request closed without merging**